CMakeLists.txt

```diff
@@ -80,6 +80,7 @@ option(VAST_ENABLE_ASSERTIONS "Enable Assertions" "${VAST_ENABLE_ASSERTIONS_DEFA
 option(VAST_RELOCATABLE_INSTALL "Enable relocatable installations" ON)
 option(ENABLE_ZEEK_TO_VAST "Build zeek-to-vast" ON)
 option(VAST_STATIC_EXECUTABLE "Link VAST statically")
+option(VAST_ENABLE_ASSERTIONS "Enable assertions" ON)
```

Comment:
What's wrong with the same option that's exposed just 4 lines above?

Reply:
Oh, interesting, I didn't notice that @tobim had already merged the same change into master two days ago when I rebased my topic branch. So I this one can just be closed now.